README.rst

```diff
@@ -0,0 +1,33 @@
+Summary
+=======
+
+Edwin is a bug management system.
+
+:source:  https://github.com/mythmon/edwin
+:issues:  https://github.com/mythmon/edwin/issues
+:license: MPLv2
+:author:  League of Assassins
+
+
+Install for hacking
+===================
+
+1. create a virtual environment
+2. run ``pip install -r requirements.txt``
+3. run ``DJANGO_CONFIGURATION=Dev ./manage.py migrate``
+4. run ``npm install``
```

Comment:
It doesn't matter what order these come in, but I'd put the npm install as step 3, so the first few steps are all about getting dependencies installed.